### PR TITLE
added nodemon dependency

### PR DESCRIPTION
**Pull request by Koyamot**
# Description

Fixes/Implements # (issue)

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Unit Tests
- [ ] Integration Tests
- [ ] User Tests

# Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -10,17 +10,17 @@ module.exports = {
...
-    'prettier/prettier': ['error', {}, { usePrettierrc: true }],
-    'no-underscore-dangle': 'off',
-    'no-shadow': 'off',
-    'no-unused-vars': 'off',
-    'object-curly-newline': 'off',
-    'import/no-named-as-default': 'off',
-    'no-use-before-define': 'off',
-    'operator-linebreak': 'off',
-  },
+  // rules: {
```
Comment (mrockingham) [inline .eslintrc.js]: If its up and running uncomment out the rules for your push

File: build/app.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ dotenv_1.default.config();
 (0, db_1.default)();
 //middleware
 const port = process.env.PORT;
+console.log(port);
```
Comment (mrockingham) [inline build/app.js]: remove console log

File: src/app.ts [hunk 3]
```diff
@@ -10,6 +10,7 @@ connectDB()
 
 //middleware
 const port = process.env.PORT
+console.log(port)
```
Comment (mrockingham) [inline src/app.ts]: remove console log

**Review by mrockingham — COMMENTED** [review]
thanks for adding nodeman...so weird. just keep an eye on your console logs

File: .eslintrc.js (hunk 1 above)
Comment (Koyamot) [reply .eslintrc.js]: Done!

File: src/app.ts (hunk 3 above)
Comment (Koyamot) [reply src/app.ts]: Done!

File: build/app.js (hunk 2 above)
Comment (Koyamot) [reply build/app.js]: Done!

**Review by mrockingham — COMMENTED** [review]
cool thanks

**Review by mrockingham — APPROVED** [review]

**Pull request merged**